SDWebImagePDFCoder/Classes/SDImagePDFCoder.m

```diff
@@ -149,7 +183,7 @@ - (UIImage *)createBitmapPDFWithData:(nonnull NSData *)data pageNumber:(NSUInteg
         return nil;
     }
     
-    CGPDFBox box = kCGPDFCropBox;
+    CGPDFBox box = kCGPDFMediaBox;
```

Comment:
Actually, we should use `mediabox` for general usage. The PDF with cropbox may contains issue when using on iOS with `imageContentEdges`. If user want, this may become a extra decoding options as well.